### PR TITLE
Fix Icon onClick, FilterSelector icons

### PR DESCRIPTION
**Pull request by ntenczar**
Icon now actually registers an onClick that works

FilterSelector uses the correct icons

### CODE REVIEW

File: lib/Icon.jsx [hunk 1]
```diff
@@ -5,11 +5,23 @@ import Tooltip from './Tooltip';
...
+    tooltip,
+    svgProps,
+    onClick
+  } = props;
   const classes = { loc: true, icon: true, muted };
 
   const theIcon = (
-    <svg {...svgProps} className={classNames(classes, className)}>
+    <svg
+      onClick={onClick}
```
Comment (meagonqz) [inline lib/Icon.jsx]: Does this produce an onclick property with a no-op function when you use the default below? Or does it not produce an onclick at all?

Comment (ntenczar) [inline lib/Icon.jsx]: onclick with noop

Comment (meagonqz) [inline lib/Icon.jsx]: why?

Comment (ntenczar) [inline lib/Icon.jsx]: no harm?

Comment (meagonqz) [inline lib/Icon.jsx]: Seems like you should not render the property if it's not set, but OK.

**Pull request merged**